packages/vuetify/src/effects/__tests__/dimensions.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { reactive } from 'vue'
+
+import {
+  dimensionsFactory,
+} from '../dimensions'
+
+describe('dimensions.ts', () => {
+  describe('dimensionsFactory', () => {
+    it('should take a single prop to use', () => {
+      const { useDimensions } = dimensionsFactory('width')
+
+      const props = { width: 100 }
+      const { dimensions } = useDimensions(props)
+
+      expect(dimensions.value.style.width).toBeDefined()
+    })
+
+    it('should take a multiple props to use', () => {
+      const { useDimensions } = dimensionsFactory('width', 'height')
+
+      const props = { width: 100, height: 200 }
+      const { dimensions } = useDimensions(props)
+
+      expect(dimensions.value.style.width).toBeDefined()
+      expect(dimensions.value.style.height).toBeDefined()
+    })
+
+    it('should default to all props when given no props', () => {
+      const { useDimensions } = dimensionsFactory()
+
+      const props = { width: 100, height: 200, maxWidth: 300, maxHeight: 400, minWidth: 500, minHeight: 600 }
+      const { dimensions } = useDimensions(props)
+
+      expect(dimensions.value.style.width).toBeDefined()
+      expect(dimensions.value.style.height).toBeDefined()
+      expect(dimensions.value.style.maxWidth).toBeDefined()
+      expect(dimensions.value.style.maxHeight).toBeDefined()
+      expect(dimensions.value.style.minWidth).toBeDefined()
+      expect(dimensions.value.style.minHeight).toBeDefined()
+    })
+  })
+
+  describe('useDimensions', () => {
+    it('should have proper styles', () => {
```

Comment:
Could be simplified to a loop to reduce vertical space

Reply:
Is this a suggestion only? I prefer to be explicit in tests.

Reply:
https://jestjs.io/docs/en/api#testeachtablename-fn-timeout

Reply:
nice, didn't know jest had support for test arguments